themes/default/content/blog/new-project-wizard-apr2024-updates/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "New Project Wizard: Org-only Templates, Default Repositories, Auto-generated Project names"
+allow_long_title: True
+date: 2024-04-19
+draft: false
+meta_desc: Pulumi Cloud adds new usability improvements that give more control to platform teams and streamline the New Project Wizard flow to deploy infrastructure 
+meta_image: meta.png
+authors:
+    - arun-loganathan
+tags:
+    - features
+    - new-project-wizard
+    - developer-portals
+    - platform-engineering
+
+---
+
+We're constantly evolving the [New Project Wizard](/docs/pulumi-cloud/developer-portals/new-project-wizard/) to make it the preferred choice for Platform teams to empower their internal teams to deploy resources quickly and in adherence to organizational standards. In January, we launched the [Developer Portal Gallery](/blog/developer-portal-gallery/) to boost development velocity and reduce operational friction.  Since then, we have seen strong adoption from our customers and received several new requests. We are excited to share the latest changes we've made. 
+
+<!--more-->
+
+## Primer on New Project Wizard and Developer Portal Gallery
+
+### The New Project Wizard (NPW)
+The NPW is a core component of [Pulumi for Developer Portal](/blog/building-developer-portals/) that guides users through a structured process to deploy infrastructure resources through Pulumi. It allows users to select from pre-defined templates or generate new programs through Pulumi AI, configure, and deploy them without leaving the browser. NPW significantly reduced the time and effort required to get projects off the ground, enabling developers to concentrate more on their development tasks and less on initial setup. Platform teams using NPW do not have to custom-build and maintain complex tooling. NPW is also a strong alternative to the [Pulumi Backstage Plugin](/docs/pulumi-cloud/developer-portals/backstage).
+
+### Developer Portal Gallery:
+The Developer Portal Gallery allows platform teams to curate and distribute approved project templates within their organization. This feature supports compliance and standardization by ensuring that all projects adhere to organizational policies and best practices right from the start. The gallery not only facilitates the discovery and use of these templates but also integrates seamlessly with the NPW, making it easy for developers to start new projects with templates that are pre-vetted and ready to deploy. 
+
+Together, the NPW and Developer Portal Gallery enhance collaboration across teams by standardizing the tools and processes used for creating and deploying projects. This alignment helps in maintaining a high level of governance as projects scale and ensures consistency across the development lifecycle. Platform teams can oversee template usage and configurations, making sure that each project conforms to the required security, architecture, and performance standards.
```

Comment:
```suggestion
Using the Developer Portal Gallery can enhance collaboration across teams by standardizing the tools and processes used for creating and deploying projects. This alignment helps in maintaining a high level of governance as projects scale and ensures consistency across the development lifecycle. Platform teams can oversee template usage and configurations, making sure that each project conforms to the required security, architecture, and performance standards.
```

Reply:
I generally think of NPW as the feature and the gallery as a page of it vs two features as they can't really be used independently.